pkg/api/v1/router_server_components.go

```diff
@@ -95,14 +102,30 @@ func (r *Router) serverComponentsCreate(c *gin.Context) {
 
 	// check server exists
 	if server == nil {
-		notFoundResponse(c, "no such server")
+		notFoundResponse(c, "server resource referenced by UUID does not exist: "+server.ID)
 		return
 	}
 
-	// components payload
+	//	// components payload
```

Comment:
can you keep the comment as it was?

Reply:
fixed in [6a44696](https://github.com/metal-toolbox/hollow-serverservice/pull/116/commits/6a44696c84d0df7fde9f35d8a9e0edac1175ace1)